html5lib/html5parser.py

```diff
@@ -1706,6 +1706,7 @@ def startTagColgroup(self, token):
 
         def startTagCol(self, token):
             self.startTagColgroup(impliedTagToken("colgroup", "StartTag"))
+            token["selfClosingAcknowledged"] = True
```

Comment:
Sorry, I'm not quite sure what you're suggesting I change here.

I based this change on other self closing tag examples. Such as:

* [`link`](https://github.com/html5lib/html5lib-python/blob/fbce67d1c3f79835b1f9eca172a1171a9df9acc0/html5lib/html5parser.py#L683-L686)
* [`meta](https://github.com/html5lib/html5lib-python/blob/fbce67d1c3f79835b1f9eca172a1171a9df9acc0/html5lib/html5parser.py#L688-L691)
* [`hr`](https://github.com/html5lib/html5lib-python/blob/fbce67d1c3f79835b1f9eca172a1171a9df9acc0/html5lib/html5parser.py#L1179-L1185)

Is `col` a different case or are you suggesting a fundamental change to all of these?

Which loop are you referring to, `mainLoop`? In this loop `new_token` is `None` by the time it checks for the `non-void-element-with-trailing-solidus` error. So I'm a bit confused.